proposals/user-scripts-api.md

```diff
@@ -54,20 +54,27 @@ User scripting related features will be exposed in a new API namespace, tentativ
 ```
 dictionary RegisteredUserScript {
   boolean? allFrames;
-  string code;
+  ScriptSource[] js;
```

Comment:
`js` goes in terms with
a. [Scripting API](https://developer.chrome.com/docs/extensions/reference/scripting/#type-RegisteredContentScript:~:text=generated%20script%20IDs.-,js,These%20are%20injected%20in%20the%20order%20they%20appear%20in%20this%20array.,-matches)
b. [Firefox user scripts API](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/register#:~:text=js,code%20to%20register.)
We are also already differentiating between `js` and `css` in `scripting.registerContentScripts`. For WASM we could use a separate parameter, since it's not a scripting language like Javascript (binary code). WDYT?

Reply:
Makes sense. `js` works for me.